libsnark/zk_proof_systems/plonk/prover.tcc

```diff
@@ -9,13 +9,13 @@
 #ifndef LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_PROVER_TCC_
 #define LIBSNARK_ZK_PROOF_SYSTEMS_PLONK_PROVER_TCC_
 
-/// Implementation of Prover interfaces for a ppzkSNARK for Plonk. See
-/// prover.hpp .
+// Implementation of Prover interfaces for a ppzkSNARK for Plonk. See
+// prover.hpp .
 
 namespace libsnark
 {
 
-/// Prover round 0 output constructor
+// Prover round 0 output constructor
```

Comment:
This was not added as part of this PR, but I think we can remove these explanations (potentially in another PR if you prefer).  It's implicit in the function signature.

Reply:
Removed.